agent/agent.go

```diff
@@ -2624,6 +2625,7 @@ func (a *Agent) addCheck(check *structs.HealthCheck, chkType *structs.CheckType,
 					return err
 				}
 				grpc.ProxyGRPC = grpcInjectAddr(grpc.GRPC, proxy.Address, port)
+				check.ExposeGRPCPort = port
```

Comment:
minor: I also think is could be clearer if the name starts with Exposed in past tense rather than Expose, since exposed ports are the product of the action Expose.Checks.

What do you think?

Reply:
I like `Exposed`! Let me change that.